### PR TITLE
Add incubation repos

### PR DESCRIPTION
**Pull request by ramceb**

### CODE REVIEW

**Comment by thilo-schmitt** [comment]
Is there a reason why signoff is disabled? I think this should be on (and is also the default for the org).

**Comment by ramceb** [comment]
> Is there a reason why signoff is disabled? I think this should be on (and is also the default for the org).

hmm. 
It was also not enabled for the main repo: https://github.com/eclipse-score/.eclipsefdn/blob/7a11a7893401a261d7c066d2be0eb8cf39464c71/otterdog/eclipse-score.jsonnet#L61

**Comment by netomi** [comment]
the org setting overrides the repo level settings anyways, its safe to remove them to get rid of the warnings.

**Comment by thilo-schmitt** [comment]
> > Is there a reason why signoff is disabled? I think this should be on (and is also the default for the org).
> 
> hmm. It was also not enabled for the main repo:
> 
> https://github.com/eclipse-score/.eclipsefdn/blob/7a11a7893401a261d7c066d2be0eb8cf39464c71/otterdog/eclipse-score.jsonnet#L61

True. I’d also strongly think we should have signoff there, too. But not part of this PR here.

**Comment by ltekieli** [comment]
@thilo-schmitt the ```web_commit_signoff_required``` setting is for organization by default set to true:
- https://otterdog.eclipse.org/projects/automotive.score#settings

The settings on the repo level can be completely dismissed.

File: otterdog/eclipse-score.jsonnet [hunk 1]
```diff
@@ -68,7 +68,7 @@ orgs.newOrg('eclipse-score') {
       code_scanning_default_setup_enabled: true,
       delete_branch_on_merge: false,
       homepage: "https://eclipse-score.github.io/inc_feo",
-      web_commit_signoff_required: no,
+      web_commit_signoff_required: false,
```
[the remark this answers is not in the document]
Comment (thilo-schmitt) [reply otterdog/eclipse-score.jsonnet]: Good solution to remove the line. Thank you!

**Review by thilo-schmitt — APPROVED** [review]

**Pull request merged**